app/models/order.rb

```diff
@@ -14,6 +14,14 @@ def quantity
     items.count
   end
 
+  def grandtotal_by_merchant(merchant_id)
+    item_orders.joins(:item).where(items: {merchant_id: merchant_id}).sum('item_orders.price * item_orders.quantity')
+  end
```

Comment:
is there a way without an argument of (merchant_id)

Reply:
That's a good question. This is most likely something we can refactor later on.